x/dclupgrade/client/cli/query_approved_upgrade.go

```diff
@@ -0,0 +1,73 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/x/dclupgrade/types"
+)
+
+func CmdListApprovedUpgrade() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list-approved-upgrade",
+		Short: "list all ApprovedUpgrade",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
+
+			pageReq, err := client.ReadPageRequest(cmd.Flags())
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			params := &types.QueryAllApprovedUpgradeRequest{
+				Pagination: pageReq,
+			}
+
+			res, err := queryClient.ApprovedUpgradeAll(context.Background(), params)
+			if err != nil {
+				return err
+			}
+
+			return clientCtx.PrintProto(res)
+		},
+	}
+
+	flags.AddPaginationFlagsToCmd(cmd, cmd.Use)
+	flags.AddQueryFlagsToCmd(cmd)
+
+	return cmd
+}
+
+func CmdShowApprovedUpgrade() *cobra.Command {
```

Comment:
Please correct the generated command to align with `CmdShowProposedUpgrade`
(check also https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/README-DEV.md#how-to-add-a-new-module)

Reply:
We use named arguments (not positional) for CLI commands (even when there is only one argument).

Reply:
fixed